packages/massa-proto-cli/README.md

```diff
@@ -0,0 +1,89 @@
+
+# Massa-Proto-CLI
+
+This package makes it easy to generate call functions for smart contract methods in TypeScrit and Assembly Script.
+The generated functions can be used in your smart contracts as well as in your projects that use typeScript or javaScript.
+
+## Usage
+
+### Install
+To get the stable version: `npm i @massalabs/massa-proto-cli`.
+
+To get the nightly version: `npm i @massalabs/massa-proto-cli`.
+
+To update the nightly version: `npm update @massalabs/massa-proto-cli`.
+
+
+### Use the CLI
+In your node project, you can run the following commands:
+ `npx massa-proto --help`
+ or
+ `npx massa-proto --addr=['your array of addresses'] --gen="mode" --out="outputDirectory"`
+
+- `--addr` must be followed by an array of contract addresses for which you want to generate your callers.
+- `--gen` must be followed by the desired generation mode: sc or web3. (sc for assembly script and web3 for typeScript generation)
```

Comment:
It is important to specify that it is not regular assembly script but rather smart contracts (for the sc mode) since the generated callers can only be used on chain via smart contracts ?